CMakeLists.txt

```diff
@@ -1029,6 +1029,13 @@ if (UNIX)
     set(HAVE_FVISIBILITY ON)
   endif (VISIBILITY)
 
+  # Check for -fno-sanitize=null
+  CHECK_C_COMPILER_FLAG("-fno-sanitize=null" HAVE_FNOSANITIZE_NULL_FLAG)
+
+  if (HAVE_FNOSANITIZE_NULL_FLAG)
+    set(HAVE_FNOSANITIZE_NULL ON)
```

Comment:
`HAVE_FNOSANITIZE_NULL_FLAG` is a bool. I thought it's better if the CMake variable we export is either defined or undefined.

Reply:
For the symbol used in code, I think we want it to be either defined or undefined. This means we cannot use the result of CHECK_C_COMPILER_FLAG directly, which is a bool. We'll need to use a second variable somewhere. So you're suggesting that I add the second var in `make/configure.cmake.h` instead of here?

Reply:
If you mean in C code, yes it has to be separate.  Having a new cmake variable here has no bearing on that: C code doesn't see this `HAVE_FNOSANITIZE_NULL` variable.  I don't see any benefit to this 2nd cmake variable.

I would expect this to be like `HAVE_TYPELIMITS_CONTROL`.

Reply:
Just a single symbol everywhere, works. Like `HAVE_TYPELIMITS_CONTROL`.